platformio.ini

```diff
@@ -72,6 +73,9 @@ monitor_filters = esp32_exception_decoder
 platform = native @ ~1.2.1
 build_flags =
     -std=c++11
+    -O0
```

Comment:
Please remove "O0".

Reply:
We do use -O0 implicitly. Showing it here made it explicit. I can remove again, but it wouldn't change anything.
 We should discuss what we really want  with native platform builds.

Reply:
Recommended O2 for native.
Please check the [mode:selected] part in Pixelix how to deal with different build configurations.

Reply:
did the mode selected method.